cmd/operator/main.go

```diff
@@ -72,6 +73,7 @@ var (
 
 func init() {
 	flag.BoolVar(&analyticsEnabled, "analytics", true, "Send analytical event (Cluster Created/Deleted etc.) to Google Analytics")
+	flag.BoolVar(&debug.DebugEnabled, "debug", false, "Enable debug logging of operator actions to a hostPath volume mounted at /var/tmp/etcd-operator/")
```

Comment:
enable-debug-file (also you can make this arg to take a actual file path. default can be `/var/tmp/etcd-operator/debug.log`)

operator probably should not know about hostpath thing. it just writes logs to somewhere.